plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/utils/AssetPackStatesFromDictionaryTest.java

```diff
@@ -23,7 +23,7 @@
 import org.junit.Test;
 
 public class AssetPackStatesFromDictionaryTest {
-  private Dictionary createDefaultTestDictionary() {
+  public static Dictionary createDefaultTestDictionary() {
```

Comment:
Changed this to public static so that we can use this helper function in PlayAssetDeliveryTest. Another way to handle this is to refactor it into a separate helper class.

Reply:
Created a separate test helper class.